README.md

```diff
@@ -12,3 +12,9 @@ use `npm run update-version -- version=[newversion]` to update the version for p
 # Linting
 
 We use [hound](https://houndci.com) to run linting on our scss files.
+
+#Deploy
+
+* Run `grunt serve` to deploy changes to the local web server. (This also adds a 'watch' so that subsequent changes are deployed automatically.)
+* Go to: //localhost:8888
```

Comment:
Can we just say here that look at the grunt serve output to see which port you need to go. This gives us some flexibility in changing ports and not having to change documentation?